_pages/en/target-solution-delivery-model-strategy-areas-of-concerns.md

```diff
@@ -388,33 +330,204 @@ This appendix lists the concerns that have been addressed and removed from this
 <br><br>
 Raised by BSIM, BOSS, Senior Advisors
 <Br><Br>
-Adding 3. Coherent set of actions / Governance / Produce standard definition of “IT Product” and “IT Solutions”.
+Adding in section "3. Coherent set of actions" the action "Governance / Produce standard definition of IT Product and IT Solutions”.
 This definition needs endorsements from more than one stakeholder.
     </td>
   </tr>
   <tr>
       <td>Enterprise Resource Planning (ERP) solutions</td>
     <td>
-How are ERP like SAP and PeopleSoft fit in this Target Model? Especially that Lines of Businesses are currently developing and testing the IT Solutions for ERP while the Strategy calls for DevOps teams to do those functions.
+How are ERPs like SAP and PeopleSoft fit in this Target Model? Especially that Lines of Businesses are currently developing and testing the IT Solutions for ERP while the Strategy calls for DevOps teams to do those functions.
 <br><br>
 Raised by BOSS
 <br><br>
-After speaking with the ERP team, they are in fact well under way and aligned with the proposed Model. They are able to do same day software changes already, though rare. They do weekly or bi-weekly changes, have different teams responsible to assure segregation of duties and capabilities (E.g. platform teams vs development team). They have worked, over the years, to obtain a level of autonomy and have demonstrated through their historical track records that this autonomy does not increases risk.
+After speaking with the ERP team, they are in fact well under way and aligned with the proposed Model.
+They are able to do same day software changes already, though it is rare.
+They do weekly or biweekly changes, have different teams responsible to assure segregation of duties and capabilities (e.g., platform team vs development team).
+They have worked, over the years, to obtain a level of autonomy and have demonstrated through their historical track records that this autonomy does not increases risk.
+    </td>
+  </tr>
+  <Tr>
+    <td>(2.1) IT Solutions that span more than one project</td>
+    <td>
+    2.1.1 Summary<br>
+<br>
+One IT solution may span more than one project and include work involved to satisfy multiple project objectives.
+For example, data management is a capability that crosses multiple project initiative and is required as part of the architecture, design, and implementation of an IT solutions.
+How to manage and keep track of such cross-project work?<br>
+<br>
+2.1.2 Raised by stakeholder(s)<br>
+- Enterprise Architecture<br>
+- Chief Data Office<br>
+<br>
+2.1.3 What is being done about this concern<br>
+<br>
+3 main aspects the strategy strives to accomplish to address this concern:<br>
+<br>
+1) Mandate Loose Coupling Architectures<br>
+2) Moving to capability-based planning<br>
+3) Measure progress by working software, not planning documents<br>
```

Comment:
3rd point below is "Rapid Elasticity" instead. Which one is the right one? ;)

Reply:
@gabrielcossette Indeed!! :)
Reworded the 3rd point to: 
Working software is much easier to produce using Cloud technologies and DevOps methodologies. In addition, working software in production can quickly react to changing conditions (such as increased load). Rapid Elasticity is part of the architecture of a solution component, especially one serving a foundational capability.
Using the cloud, virtual instances of solution components can be spun up within milliseconds automatically based on increased demands.
To use the cloud properly, solution component must be architected properly.

Reply:
FYI, this section are the "areas of concerns" that are believed to be solved. And we explain how they are solved.

Reply:
Understood.
Great! I made some tweaks as well (in bold). Feel free to adjust.

> **It's more meaningful to measure progress through working software.**
> **It's also** much easier to produce working software using Cloud technologies and DevOps methodologies.
> **This way,** working software in production can quickly react to changing conditions (such as increased load).
> Rapid Elasticity is part of the architecture of a solution component, especially one serving a foundational capability.
> Using the cloud, virtual instances of solution components can be spun up within milliseconds automatically based on increased demands.
> **But** to use the cloud properly, solution component must be architected properly.

Reply:
And very interesting and relevant to document how each concerns have been solved, bravo! 👍